packages/hash/api/src/graphql/resolvers/entity/linkGroups.ts

```diff
@@ -35,8 +35,10 @@ export const linkHasSameDestinationAs = (linkA: Link) => (linkB: Link) =>
 const addEntityOutgoingLinks =
   (client: DbClient, linkGroupsObject: LinkGroupsObject, depth: number) =>
   async (entity: Entity) => {
+    const { accountId, entityId, entityVersionId } = entity;
+
     const existingGroups =
-      linkGroupsObject?.[entity.entityId]?.[entity.entityVersionId];
+      linkGroupsObject?.[`${accountId}*${entityId}`]?.[entityVersionId];
```

Comment:
We need the `accountId` in this intermediate object to be able to add it to the API output. We need it in the API output to rewrite the `sourceEntityId` to contain the `accountId` before sending it to blocks.